config/config.js

```diff
@@ -4,10 +4,11 @@ export default {
   // Note: typeof(process.env.MOCK) === string
   define: {
     MOCK_UI: process.env.MOCK === 'true',
+    ENDPOINT: 'http://localhost:8000/api/v1/endpoints',
```

Comment:
Now this is broken again! We shouldn't be changing the "real" `config.js` for local testing. I'd assumed that was the point of your `.gitignore` changes, where you'd have alternate unstaged config files *without* changing the real version. Someone's going to end up accidentally committing this (as you've already done) and breaking the dashboard; we really don't want that to happen.